### ISSUE TITLE
hmr and watcher tests failing randomly

### ISSUE THREAD

**Issue by oliger** [parcel-bundler/parcel#52]
While working on #50 I had those two tests failing randomly... I did not took the time to understand the issue... Maybe it come from my machine.

```
 // ...

  hmr
    ✓ should emit an HMR update for the file that changed (244ms)
    ✓ should emit an HMR update for all new dependencies along with the changed file (309ms)
Assertion failed: (0), function uv_close, file ../deps/uv/src/unix/core.c, line 182.
error Command failed with signal "SIGABRT".
```

```
  // ...

  watcher
    ✓ should rebuild on file change (223ms)
    ✓ should re-generate bundle tree when files change (319ms)
    1) should only re-package bundles that changed
    ✓ should unload assets that are orphaned (295ms)


  60 passing (10s)
  1 failing

  1) watcher should only re-package bundles that changed:

      AssertionError [ERR_ASSERTION]: 1512573179 != 1512573179
      + expected - actual


      at Context.<anonymous> (test/watcher.js:100:12)
      at <anonymous>
```

**Comment by thejameskyle** [comment]
I've been running into the same problems. I've been looking into them, it seems the uv_close issue is related to sass and if you skip the sass tests you won't get that error anymore

**Comment by albinotonnina** [comment]
Same here

**Comment by devongovett** [comment]
I think the uv_close issue might be https://github.com/paulmillr/chokidar/issues/612

**Comment by devongovett** [comment]
Worked around this issue by disabling FS events on macOS for the tests. Other general test fixes here as well: #74
